src/main/java/seedu/address/logic/parser/ModifyCommandParser.java

```diff
@@ -59,10 +63,7 @@ public ModifyCommand parse(String args) throws ParseException {
         }
         if (argMultimap.getValue(PREFIX_DEADLINE).isPresent()) {
             editInternshipDescriptor.setDeadline(ParserUtil
-                    .parseDeadline(
-                            argMultimap.getValue(PREFIX_DEADLINE).get(),
-                            argMultimap.getValue(PREFIX_START_DATE).get()
-                    ));
+                    .parseDeadline(argMultimap.getValue(PREFIX_DEADLINE).get()));
```

Comment:
Perhaps can do orElse if want to avoid error with null value when passed to parseDeadline

Reply:
the previous if clause already check for the case where PREFIX_DEADLINE is null